ddtrace/settings/_database_monitoring.py

```diff
@@ -57,4 +57,9 @@ def _get_dbm_comment(db_span):
         db_span.set_tag_str(DBM_TRACE_INJECTED_TAG, "true")
         dbm_tags[DBM_TRACE_PARENT_KEY] = db_span.context._traceparent
 
-    return _generate_sql_comment(**dbm_tags)
+    sql_comment = _generate_sql_comment(**dbm_tags)
+    if sql_comment and sql_comment[0] == " " and sql_comment[-1] != " ":
```

Comment:
do you need to check these? if we have one then we know for sure it has a leading whitespace because that is what the vendored code does?

we can probably just shorten this all to:

```python
if sql_comment:
    return sql_comment.strip() + " "
return None    # Is this correct? I also wonder if this is more explicit than `return sql_comment` ... not sure, just where my mind went
```